packages/core/admin/admin/src/pages/Settings/components/Tokens/TokenBox.tsx

```diff
@@ -35,54 +38,67 @@ export const TokenBox = ({ token, tokenType }: TokenBoxProps) => {
     }
   };
 
+  const getGuidedTourTooltip = (tokenType: string) => {
+    switch (tokenType) {
+      case 'api-token':
+        return unstable_tours.apiTokens.CopyAPIToken;
```

Comment:
Do you need the getter function here? Couldn't you just use unstable_tours.apiToken.copyAPIToken directly?